content/en/actions/app_builder/build.md

```diff
@@ -16,48 +16,102 @@ further_reading:
 
 You can create an app or edit existing apps from the [App Builder][1] page. The page lists information about existing apps, including the following:
 - Author
-- Status
-- Date that each app was last modified
+- Tags
+- Date that the app was last modified
 - Whether the app is published
```

Comment:
```suggestion
- Date that each app was last modified
- Whether each app is published
```
"Each" makes sense to me here in both cases, because it covers a high-level list of all of your apps rather than a single one.